### PR TITLE
add: Rating Component

### PR DESCRIPTION
**Pull request by kavyakunder**
Rating Component with 5 stars

rating1

### CODE REVIEW

File: Components/Rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                <i class="fab fa-github fass"></i
+              ></a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <div class="wrapper">
```
Comment (junaidshaikh-js) [inline Components/Rating/rating.html]: can be wrapped inside `main` element

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

**Review by shuklahimanshu81 — COMMENTED** [review]
Looks good to me. Can be merged.

File: Components/Rating/rating.html (hunk 1 above)
Comment (kavyakunder) [reply Components/Rating/rating.html]: done!

**Pull request merged**